src/DetailsView/reports/components/report-sections/report-section-factory.tsx

```diff
@@ -11,6 +11,7 @@ export type SectionProps = {
     scanDate: Date;
     environmentInfo: EnvironmentInfo;
     scanResult: ScanResults;
+    utcDateConverter: (date: Date) => string;
```

Comment:
It looks like the name give some space for interpretation, maybe `toUtcString` ??